test/autorest/generated/lrogroup/pollers_helper.go

```diff
@@ -87,6 +87,12 @@ type pollingTracker interface {
 
 	// converts an *azcore.Response to an error
 	handleError(resp *azcore.Response) error
+
+	// returns the desired configuration for the final GET URI that overrides the value set in the finalGetURL
+	finalStateVia() string
```

Comment:
Looks like this method is referenced anywhere, can it be removed?

Reply:
Yes it can, will remove it.

Reply:
Done